libpod/container_internal.go

```diff
@@ -909,8 +936,10 @@ func (c *Container) postDeleteHooks(ctx context.Context) (err error) {
 
 // Make standard bind mounts to include in the container
 func (c *Container) makeBindMounts() error {
-	if err := os.Chown(c.state.RunDir, c.RootUID(), c.RootGID()); err != nil {
-		return errors.Wrapf(err, "error chown %s", c.state.RunDir)
+	if os.Getuid() == 0 {
+		if err := os.Chown(c.state.RunDir, c.RootUID(), c.RootGID()); err != nil {
+			return errors.Wrapf(err, "error chown %s", c.state.RunDir)
```

Comment:
Better error message here?

Reply:
I have only changed the indentation.  I'll change the error message as part of the patch